brayovsky/pivotaly
# Bundle App

#### What does this PR do?
Bundles production app to one file 

#### How should this be manually tested?
Run `npm run dev`. The app should be bundled to a `dist` folder and should run well while debugging in development.

#### Any background context you want to provide?
Bundling the app while greatly reduce the app size and make downloading and installing the extension much faster.

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
